_security-analytics/threat-intelligence/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Threat intelligence
+nav_order: 40
+has_children: true
+---
+
+# Threat intelligence
+
+You can use the threat intelligence feature in OpenSearch to integrate your threat intelligence feeds. Feeds are comprised of Indicators of Compromise (IOCs), which search for malicious indicators in your data. Through monitors, findings are generated, and notifications are sent when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.
+
+To access threat intelligence, log into OpenSearch Dashboards, and select **Security Analytics** > **Threat Intelligence**.
+
+You can interact with threat intelligence in the following ways:
+
+- [Threat intelligence API]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/)
+- [OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/getting-started/)
```

Comment:
This seems a bit odd to link to the Getting Started topic. Perhaps we can do something like this, 
```suggestion
- OpenSearch Dashboards. To configure threat intelligence through the OpenSearch Dashboards interface, see [Getting Started]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/getting-started/).
```